apps/angular/src/core/interceptors/auth.interceptor.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import {
+  HttpEvent,
+  HttpInterceptor,
+  HttpHandler,
+  HttpRequest,
+} from '@angular/common/http';
+import { Observable, switchMap, map } from 'rxjs';
+
+import { JwtService } from '../services/';
+
+/** Api key interceptor. */
+@Injectable()
+export class AuthInterceptor implements HttpInterceptor {
+  public constructor(private readonly jwtService: JwtService) {}
+
+  /**
+   * Interceptor.
+   * @param req HttpRequest object.
+   * @param next HttpHandler function.
```

Comment:
Please don't just repeat variable names 
```suggestion
   * @param req Request object.
   * @param next Handler function.
```